CHANGELOG.md

```diff
@@ -24,7 +27,7 @@
 
 ## 1.2.2 (2019-02-13)
 
-* Fix a BC breaking change for PHP 5.6/PHPUnit 5.7.27 (#947) 
```

Comment:
My editor automatically trims trailing whitespace and I figured it wouldn't hurt to commit these changes.